site/content/docs/main/resource-filtering.md

```diff
@@ -303,8 +312,55 @@ The policies YAML config file would look like this:
       action:
         type: skip
     ```
+### IncludeExcludePolicy
+The `includeExcludePolicy` is used to filter resources based on the namespace-scoped and cluster-scoped resources. User can use it 
+to define a group of filters and reuse them across different backups.
+
+For example, user can configmap `my-policy` of resource policies with following content:
```

Comment:
It would be better if we could give a valid ConfigMap as an example.

``` yaml
apiVersion: v1
data:
  my-policy.yaml: |
    version: v1
    includeExcludePolicy:
      includedClusterScopedResources:
        - "crd"
      excludedClusterScopedResources: []
      includedNamespaceScopedResources: []
      excludedNamespaceScopedResources:
        - "configmap"
        - "event"
kind: ConfigMap
metadata:
  name: my-policy
  namespace: velero
```

Reply:
But normally the flow is to write the yaml file and create a configmap via the yaml file, instead of creating a configmap directly, no?